build-system/common/check-package-manager.js

```diff
@@ -217,10 +217,10 @@ function logNpmVersion() {
  */
 function checkPythonVersion() {
   // Python 2.7 is EOL but still supported
-  // Python 3.5+ are still supported (TODO: deprecate 3.5 on 2020-09-13)
+  // Python 3.6+ are still supported
   // https://devguide.python.org/#status-of-python-branches
-  const recommendedVersion = '2.7 or 3.5+';
-  const recommendedVersionRegex = /^2\.7|^3\.[5-9]/;
+  const recommendedVersion = '2.7 or 3.6+';
+  const recommendedVersionRegex = /^2\.7|^3\.[6-9]/;
```

Comment:
The next version of Python will be 3.10, if you want to be able to handle that starting now:
`/^2\.7|^3\.(?:[6-9]|1\d)/`

Reply:
Nice catch. I've included your suggestion and tested it. (Next time this needs to be changed is when 3.19 gives way to 3.20.)

Reply:
I think we'll be fine for a while